src/main/java/duke/util/Ui.java

```diff
@@ -0,0 +1,118 @@
+package duke.util;
+
+import duke.task.Task;
+
+import java.util.List;
+
+public class Ui {
+
+    /** Final value for the border */
+    final private static String LINES = "===============================================";
+
+    /**
+     * Basic constructor
+     */
+    public Ui (){}
```

Comment:
Is this redundant as the default constructor is automatically created? Is there a space in between the closing bracket `)` and `{`